android/build.gradle

```diff
@@ -1,12 +1,12 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.6.10'
     repositories {
         google()
         mavenCentral()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.1.0'
+        classpath 'com.android.tools.build:gradle:7.1.2'
```

Comment:
I have no idea why my original project setup didn't have the latest stuff. Maybe my dependencies changed, and so recreating the Android folder at this point gives different values.